style/_dropdown.scss

```diff
@@ -29,17 +29,14 @@ details.sd-dropdown {
       display: none;
     }
 
-    &:focus {
-      outline: none;
-    }
-
```

Comment:
As mentioned in the PR description, I really don't understand why this rule was ever put in place.